internal/utils/utils.go

```diff
@@ -67,7 +67,7 @@ const (
 func ExecKubectlCommand(ctx context.Context, mode ExecMode, kubeConfigPath string, kubeContext string, args ...string) (string, error) {
 	var stdoutBuf, stderrBuf bytes.Buffer
 
-	if kubeConfigPath != "" {
+	if kubeConfigPath != "" && !containsMultipleConfigFiles(kubeConfigPath){
```

Comment:
(While not chimed in on details on why we pass an additional config to `kubectl` here)

You likely want to make use of [`filepath.SplitList`](https://golang.org/pkg/path/filepath/#SplitList) and simply check the `len`, as this already deals with `runtime.GOOS` / multi-path strings for you.